src/System/Wlog/LoggerConfig.hs

```diff
@@ -351,3 +354,24 @@ maybeLogsDirB prefix = mempty { _lcLogsDirectory = prefix }
 -- | Setup 'lcLogsDirectory' inside 'LoggerConfig' to specific prefix.
 logsDirB :: FilePath -> LoggerConfig
 logsDirB = maybeLogsDirB . Just
+
+-- | Lens to help to change some particular logger properties.
+--
+-- For example if you want to use default configurations,
+-- but need to change logger's severity to 'warningPlus'
+-- you can do it this way:
+--
+-- @
+-- 'System.Wlog.Launcher.launchWithConfig'
+--     ( defaultConfig "myLogger"
+--     & 'atLogger' "myLogger"
+--     . 'ltSeverity' ?~ 'warningPlus' )
+--     "myLogger"
+-- @
+--
+atLogger :: LoggerName -> Traversal' LoggerConfig LoggerTree
+atLogger logName = lcTree . leveldown (LoggerName <$> Text.splitOn "." (getLoggerName logName))
+  where
+    leveldown []     = error "Logger should be provided"
+    leveldown [x]    = ltSubloggers . at x . _Just
+    leveldown (x:xs) = ltSubloggers . at x . _Just . leveldown xs
```

Comment:
Could you add type to this function? Maybe `-XPartialSignatures` can help you...